packages/graphql-hooks/index.d.ts

```diff
@@ -105,23 +111,26 @@ interface UseQueryOptions extends UseClientRequestOptions {
   ssr?: boolean
 }
 
-interface UseClientRequestResult {
+interface UseClientRequestResult<Response> {
```

Comment:
@good-idea Any reason we're using `ResponseData` as the generic name in other places, but `Response` here?

Reply:
Good catch, thanks!